src/components/templates/top/styles.module.scss

```
@@ -0,0 +1,27 @@
+@use '../../../styles/mixin.scss';
+@use '../../../styles/variables.module.scss' as variables;
+
+.category_container {
+  background-color: inherit;
+  margin-left: 1.125rem;
+  display: flex;
+  flex-direction: column;
+  row-gap: 1rem;
+  @include mixin.tb {
+    margin-left: 2rem;
+  }
+  @include mixin.pc {
+    margin-left: 4rem;
+  }
+}
+
+.category_card_container {
+  display: flex;
+  overflow: scroll;
+  column-gap: 0.5rem;
+  @include mixin.pc {
+    flex-wrap: wrap;
+    column-gap: 1rem;
+    row-gap: 1.5rem;
+  }
+}
```

⚙️ [nits]
カードがはみ出る時の処理ですが、要件としては
- スマホ（デフォルト）
  - X 軸方面をスクロール可能
- PC 
  - スクロール不可

としたいと思うので、次のように素直に実装してみた方が良いかなと思いました！

```suggestion
.category_card_container {
  display: flex;
  overflow-x: auto;
  column-gap: 0.5rem;
  @include mixin.pc {
    overflow: unset;
    flex-wrap: wrap;
    column-gap: 1rem;
    row-gap: 1.5rem;
  }
}
```